sklearn/linear_model/tests/test_bayes.py

```diff
@@ -6,6 +6,7 @@
 from math import log
 
 import numpy as np
+import pandas as pd
```

Comment:
`pandas` is a soft dependency in scikit-learn.

We use `pd = pytest.importorskip("pandas")` in the test itself. It will be skipped if `pandas` is not installed that is completely fine.